components/brave_wallet/browser/blockchain_registry.cc

```diff
@@ -100,26 +100,55 @@ void BlockchainRegistry::GetAllTokens(const std::string& chain_id,
   std::move(callback).Run(std::move(tokens_copy));
 }
 
-void BlockchainRegistry::GetBuyTokens(const std::string& chain_id,
+void BlockchainRegistry::GetBuyTokens(mojom::OnRampProvider provider,
+                                      const std::string& chain_id,
                                       GetBuyTokensCallback callback) {
   std::vector<brave_wallet::mojom::BlockchainTokenPtr> blockchain_buy_tokens;
-  for (auto token : *kBuyTokens) {
+  std::vector<mojom::BlockchainToken> buy_tokens;
```

Comment:
`std::vector<mojom::BlockchainToken>* buy_tokens = nullptr;` better select vector by pointer here, so it is not copied.

Reply:
Fixed in fdfd2f5b0d086b4d433080c750993185e3f35f21.